sympy/functions/elementary/trigonometric.py

```diff
@@ -197,6 +197,31 @@ def _pi_coeff(arg, cycles=1):
             return cx
 
 
+def _expand_sin_cos_Add(args):
+    """
+    Expands (sin(Add(*args)), cos(Add(*args))) more efficiently 
+    than recursively expanding two terms at a time
+
+    Examples
+    ========
+
+    >>> from sympy.functions.elementary.trigonometric import _expand_sin_cos_Add as expand_sc
+    >>> from sympy.abc import x,y,z
+    >>> expand_sc(x,y,z)
+    >>> expand_sc([x,y,z])
+    ((-sin(x)*sin(y) + cos(x)*cos(y))*sin(z) + (sin(x)*cos(y) + sin(y)*cos(x))*cos(z), (-sin(x)*sin(y) + cos(x)*cos(y))*cos(z) - (sin(x)*cos(y) + sin(y)*cos(x))*sin(z))
```

Comment:
It is generally not a good idea to have doctests for internal functions, you can probably add them to `test_trigonometric.py`.